easybuild/easyconfigs/b/bpp-phyl/bpp-phyl-2.4.1-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'CMakeMake'
+
+name = "bpp-phyl"
+version = '2.4.1'
+
+homepage = 'https://github.com/BioPP/bpp-phyl'
+description = """Bio++ Phylogenetic Library"""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+github_account = 'BioPP'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['e7bf7d4570f756b7773904ffa600ffcd77c965553ddb5cbc252092d1da962ff2']
+
+builddependencies = [('CMake', '3.13.3')]
+
+dependencies = [
+    ('bpp-core', '2.4.1'),
+    ('bpp-seq', '2.4.1'),
+]
+
+configopts = '-Dbpp-core_DIR=${EBROOTBPPMINCORE}/lib64/cmake/bpp-core/ '
+configopts += '-Dbpp-seq_DIR=${EBROOTBPPMINSEQ}/lib64/cmake/bpp-seq/ '
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["lib64"]
```

Comment:
@omula I would make this a bit more specific:

```python
sanity_check_paths = {
    'files': ['lib64/libbpp-phyl.a', 'lib64/libbpp-phyl.%s' % SHLIB_EXT],
    'dirs': ['include/Bpp']
}

Reply:
done